content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Manage billing with consumption limits
+subtitle: Learn how to set usage quotas per project with the Neon API
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2023-11-03T15:15:02.548Z'
+---
+
+When setting up your billing solution with Neon, you may want to impose some hard limits on how much storage or compute size a given project can consume. For example, you may want to cap how much usage your free tier users can consume versus pro or enterprise users. With the Neon API, you can use the `quota` key to set usage limits for a variety of consumption metrics. These limits act as thresholds after which all active computes for a project are [suspended](#what-happens-when-the-quota-is-met). 
+
+## Metrics and quotas
+By default, Neon tracks a variety of consumption metrics at the project level. If you want to set quotas (max limits) for these metrics, you need to explicitly [configure](#configuring-quotas) them. 
+
+### Available metrics
+
+Here are the relevant metrics that you can set project-level quotas for:
+
+* `active_time_seconds`
+* `compute_time_seconds`
+* `written_data_bytes`
+* `data_transfer_bytes` 
+
+These consumption metrics represent total cumulated usage across all branches and computes in a given project, accrued so far in a given monthly billing period. They are refreshed on a set day every month, on whichever date your new billing period starts. 
+
+To find the current usage level for any of these metrics, see [retrieving details about a project](#retrieving-details-about-a-project). You can read more about these metrics and how they impact billing [here](/docs/billing).
+
+### Corresponding quotas
+
+You can set quotas for these consumption metrics per project using the following Neon API URL:
+
+```bash
+/projects/{project_id}/settings/quota
+```
+
+The `quota` object includes the array of parameters used to set threshold limits. Their names generally match their corresponding metric:
+
+* `active_time_seconds` &#8212; Sets the maximum amount of wall-clock time allowed in total across all of a project's compute endpoints. This means the total elapsed time, in seconds, from start to finish for each transaction handled by the project's endpoints, accumulated before the one-month refresh date.
+* `compute_time_seconds` &#8212; Sets the maximum amount of CPU seconds allowed in total across all of a project's compute endpoints. This differs from `active_time_seconds` in that it only counts the time the CPU spends executing a process. It excludes time spent waiting for external resources, I/O operations, or time spent in a blocked or idle state.
```

Comment:
I am not sure about "This differs from `active_time_seconds` in that it only counts the time the CPU spends executing a process. It excludes time spent waiting for external resources, I/O operations, or time spent in a blocked or idle state."

The way I understand it is this, but maybe I am wrong:

If a compute with 4 vCPUs runs for 1 second, active_time_seconds=1.
If a compute with 4 vCPUs runs for 1 second, compute_time_seconds=4.

So, the larger your computes, the faster you'll burn through compute_time_seconds.

Reply:
I've revised it to:

Sets the maximum amount of CPU seconds allowed in total across all of a project's compute endpoints, including any endpoints deleted during the current billing period. Note that the larger your compute endpoints, the faster you'll consume through `compute_time_seconds`. For example, a compute instance with 0.25 vCPU uses 0.25 seconds of compute time for every second it's active. If it has 4 vCPUs, it uses 4 seconds of compute time in one active second.
   | vCPUs  | `active_time_seconds` | `compute_time_seconds` |
   |--------|-----------------------|------------------------|
   | 0.25   | 1                     | 0.25                   |
   | 4      | 1                     | 4                      |